rskj-core/src/integrationTest/resources/integration-test-rskj.conf

```diff
@@ -0,0 +1,33 @@
+peer {
+
+    discovery = {
+
+        # if peer discovery is off
+        # the peer window will show
+        # only what retrieved by active
+        # peer [true/false]
+        enabled = true
+
+        # List of the peers to start
+        # the search of the online peers
+        # values: [ip:port]
+        ip.list = [
```

Comment:
is there a need to use mainnet bootstraps in the int. tests? Can this list, for instance, be empty?

Reply:
Addressed